examples/Example/src/new_api/camera/index.tsx

```diff
@@ -0,0 +1,351 @@
+  const [recordingIntervalHandle, setRecordingIntervalHandle] = useState(-1);
+
+  const filtersPanGesture = Gesture.pan()
+    .setOnUpdate((e) => {
+      'worklet';
+      filter.value =
+        filter.value + (filterOffset.value - e.translationX) * 0.01;
+      filterOffset.value = e.translationX;
+
+      runOnJS(updateSelectedFilter)();
+    })
+    .setOnEnd(() => {
+      'worklet';
+      filterOffset.value = 0;
+      runOnJS(stopFilterScroll)();
+    });
+
+  const buttonTapGesture = Gesture.tap().setOnEnd((e, success) => {
+    'worklet';
+    if (success) {
+      runOnJS(takePhoto)();
+    }
+  });
+
+  const buttonDoubleTapGesture = Gesture.tap()
+    .setTapCount(2)
+    .setOnEnd((_e, success) => {
+      'worklet';
+      if (success) {
+        runOnJS(takeSeries)();
+      }
+    });
+
+  const buttonPanGesture = Gesture.pan()
+    .addSimultaneousGesture(filtersPanGesture)
+    .setOnUpdate((e) => {
+      'worklet';
+      if (isRecording) {
+        if (e.velocityY < 0) {
+          zoom.value *= 1.05;
+        } else if (e.velocityY > 0) {
+          zoom.value *= 0.95;
+        }
+      }
+    })
+    .setOnEnd(() => {
+      'worklet';
+      if (isRecording) {
+        runOnJS(finishRecording)();
+      }
+    });
+
+  const buttonLongPressGesture = Gesture.longPress().setOnStart(() => {
+    'worklet';
+    runOnJS(startRecording)();
+  });
+
+  const previewPinchGesture = Gesture.pinch()
+    .setOnStart(() => {
+      'worklet';
+      runOnJS(setScale)(zoom.value);
+    })
+    .setOnUpdate((e) => {
+      'worklet';
+      zoom.value = scale * e.scale;
+    });
+
+  const panGestureHandler = useAnimatedGesture(filtersPanGesture);
+
+  const buttonGestureHandler = useAnimatedGesture(
+    buttonTapGesture
+      .requireToFail(buttonDoubleTapGesture)
+      .simultaneousWith(buttonPanGesture)
+      .simultaneousWith(buttonLongPressGesture)
+  );
+
+  const scaleGestureHandler = useAnimatedGesture(previewPinchGesture);
+
+  function stopFilterScroll() {
+    filter.value = withTiming(updateSelectedFilter(), { duration: 200 });
+  }
+
+  function updateSelectedFilter() {
+    const selectedFilter = Math.round(
+      Math.min(filters.length - 1, Math.max(filter.value, 0))
+    );
+    setSelectedFilter(selectedFilter);
+
+    return selectedFilter;
+  }
+
+  function takePhoto() {
+    Alert.alert('You took a photo');
+  }
+
+  function takeSeries() {
+    Alert.alert('You took a series of photos');
+  }
+
+  function startRecording() {
+    setIsRecording(true);
+    setRemainingTime(MAX_VIDEO_DURATION);
+    setRecordingIntervalHandle(
+      setInterval(() => {
+        setRemainingTime((r) => r - 200);
+      }, 200)
+    );
+  }
+
+  function finishRecording() {
+    setIsRecording(false);
+    clearInterval(recordingIntervalHandle);
+    setRemainingTime(MAX_VIDEO_DURATION);
+
+    Alert.alert(
+      `You took a video (${(MAX_VIDEO_DURATION - remainingTime) * 0.001} s)`
```

Comment:
```suggestion
      `You took a video (${(MAX_VIDEO_DURATION - remainingTime) / 1000} s)`
```

Reply:
Done in c9c43d8757b585808251376d16f6a0182c2842f3